iOS/Projects/Features/SignUp/Sources/Presentation/SignUpProfileScene/View/NickNameDuplicatingCheckerView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NickNameDuplicatingCheckerView.swift
+//  SignUpFeature
+//
+//  Created by 안종표 on 12/11/23.
+//  Copyright © 2023 kr.codesquad.boostcamp8. All rights reserved.
+//
+
+import Foundation
+
+final class NickNameDuplicatingCheckerView: CheckerView {
+  override init(frame _: CGRect) {
+    super.init(frame: .zero)
+  }
```

Comment:
P3: 흐름에 맞게 frame을 그대로 넣어도 좋을 것 같아요.
```suggestion
  override init(frame: CGRect) {
    super.init(frame: frame)
  }
```